hono/seed.ts

```diff
@@ -1,15 +1,32 @@
+import { parseArgs } from 'util';
 import { db } from './db';
-import { students } from './family/schema';
 import { meta } from './admin/schema';
+import { students } from './family/schema';
+
+const {
+  values: { push }
+} = parseArgs({
+  args: Bun.argv,
+  options: {
+    push: {
+      type: 'boolean',
+      default: false
+    }
+  },
+  strict: true,
+  allowPositionals: true
+});
 
 const now = new Date();
 const academicYear =
   now.getFullYear() - Math.floor(now.getFullYear() / 100) * 100;
```

Comment:
academic year same thing 
```suggestion
  const now = new Date();
const year = now.getFullYear();
const month = now.getMonth() + 1; 
const academicYearStart = month >= 10 ? year : year - 1;
const academicYear = academicYearStart % 100; 
```